src/components/contexts/filter.js

```diff
@@ -87,6 +87,17 @@ export function FilterProvider({ children }) {
             showToast(getText('filter.error.getData'));
         }
     };
+
+    const resetFilter = () => (
+        setFilterData({
+            start: DayJS().startOf('month').toDate(),
+            end: DayJS().endOf('month').toDate(),
+            dayoffTypes: [],
+            slackUsers: [],
+            status: []
```

Comment:
The default values of filter data are already declared in the context. They should be declared only one time and re-used.